extensions/sat/README.md

```diff
@@ -11,24 +11,41 @@ This document explains the fields of the Satellite Extension to a STAC Item. Sat
 The Satellite extension requires the [Instrument Fields](../../item-spec/common-metadata.md#instrument).
 
 - [Example (Landsat 8)](examples/example-landsat8.json)
+- [Example (Sentinel 1)](examples/example-sentinel1.json)
 - [JSON Schema](json-schema/schema.json)
 
 ## Item fields
 
 | Field Name       | Type                     | Description |
 | ---------------- | ------------------------ | ----------- |
+| sat:platform_serial_identifier | string | The International Designator, also known as COSPAR ID, and NSSDCA ID |
 | sat:orbit_state        | string        | The state of the orbit. Either `ascending` or `descending` for polar orbiting satellites, or `geostationary` for geosynchronous satellites |
+| sat:absolute_orbit     | integer       | The obsolute orbit number at the time of acquisition. |
```

Comment:
We've discussed absolute_orbit earlier in issues and PRs and decided against it as it doesn't seem very useful for search. So I'd remove this again unless there are new strong arguments for it.

Reply:
I would have several reasons:

1.  It is a very common parameter in the industry. Often used in the items product identifier thus derivable for search.
2.  In mission planning and tasking, the absolute orbit is often use as a reference. Especially in the non systematic acquisition missions. The resulting product can be tagged with the absolute orbit and thus searchable as such.
3.  In the case of orbital changes during the mission modifying the ground track and thus the repeat cycle and thus the number or relative orbits, the combination of cycle and relative orbit is not sufficient to derive an absolute orbit. This has been the case for ENVISAT mission: To ensure an additional 3 years lifespan, the ENVISAT satellite moved to a new lower orbit on October  22th, 2010. From November 2nd, 2010, for the ENVISAT extension orbit, the ground track changes and consequently the repeat cycle changes: 30 days with 431 orbits per cycle instead of 35 days-501 orbits per cycle.

Reply:
So would users use the absolute orbit for that? It seems like using datetime would be a more natural way to do that. Users would know the dates of the changes and if they corresponded with their dates of interest. Figuring out absolute orbit seems like an unnecessary step.

I'm not opposed to including it, it would be an optional field and I have worked with some datasets that use it explicitly in data organization.

Reply:
I think I'm ok with this in the 'sat extension'. I think we should aim to consolidate fields that are generally useful for satellite data providers, even if they wouldn't be in search. Instead of making providers all come up with their own they should be able to use common ones. We could make some additional extension that is just focused on 'additional satellite properties', but that seems like overkill to me. I feel ok with it here since it's an extension, and an optional field. 

Anything in core should be for search, absolutely, but I think it's ok for extensions to add in fields that aren't strictly for search. Like projection & rendering hints.